assets/src/components/pr/scm/CreateScmWebhook.tsx

```diff
@@ -108,75 +100,84 @@ export function CreateScmWebhookModalBase({
             secondary
             onClick={() => onClose?.()}
           >
-            {success ? 'Close' : 'Cancel'}
+            {newWebHook || error ? 'Close' : 'Cancel'}
           </Button>
         </div>
       }
     >
-      {success ? (
-        <Body1P>Successfully created webhook for {connection.name}</Body1P>
-      ) : (
-        <div
-          css={{
-            display: 'flex',
-            flexDirection: 'column',
-            gap: theme.spacing.medium,
-          }}
-        >
-          <ScmWebhookForm
-            {...{
-              type: 'create',
-              connection,
-              formState,
-              updateFormState,
-              error,
-            }}
+      <Flex
+        flexDirection="column"
+        gap="large"
+      >
+        {error && (
+          <GqlError
+            header="Something went wrong"
+            error={error}
           />
-          {error && <GqlError error={error} />}
-        </div>
-      )}
+        )}
+        {newWebHook && !error && (
+          <>
+            <FormField label="Webhook URL">
```

Comment:
Could we add a bit of text field here that says:

 "Add a new webhook in your source control provider with the following url and validation secret"